engine/src/main/java/org/camunda/bpm/engine/impl/optimize/OptimizeManager.java

```diff
@@ -121,9 +121,12 @@ public void fetchHistoricVariableUpdateByteArrays(List<String> byteArrayIds) {
     checkIsAuthorizedToReadHistoryAndTenants();
 
     String[] operationTypes = new String[]{
-      UserOperationLogEntry.OPERATION_TYPE_ASSIGN,
-      UserOperationLogEntry.OPERATION_TYPE_CLAIM,
-      UserOperationLogEntry.OPERATION_TYPE_COMPLETE};
+      UserOperationLogEntry.OPERATION_TYPE_SUSPEND_JOB,
+      UserOperationLogEntry.OPERATION_TYPE_ACTIVATE_JOB,
```

Comment:
There's also the operation type `UserOperationLogEntry.OPERATION_TYPE_SUSPEND_JOB_DEFINITION`. When suspending a job definition, a user can also choose to include all jobs of that definition. Should these operations be exported to Optimize as well?

Reply:
I don't think that this is necessary. But just to be sure: would it be possible that a job suspension results in a process instance suspension as well?

Reply:
No, that's not possible. I think part of my question was that I wondered why `OPERATION_TYPE_SUSPEND_JOB` is included but `OPERATION_TYPE_SUSPEND_JOB_DEFINITION`. But the tests explain that `OPERATION_TYPE_SUSPEND_JOB` is needed in the case of batches.

Reply:
Yes, exactly. Then I would keep it as it is.